File: wronggou-manager-web/src/main/resources/static/js/service/ContentService.js
app.service('ContentService',function($http){

    this.findPage=function(page, rows){
        return  $http.get('http://localhost:8086/content/findPage?page=' + page + '&rows=' + rows);
    }
    this.save=function(entity){
        return   $http.post('http://localhost:8086/content/add',entity);
    }
    this.update=function(entity){
        return   $http.post('http://localhost:8086/content/update', entity);
    }
    this.findOne=function(id){
        return $http.get('http://localhost:8086/content/findOne?id='+id);
    }
    this.delete=function(selectIds){
        return  $http.get('http://localhost:8086/content/delete?ids='+selectIds);
    }

});

File: wronggou-content-interface/src/main/java/com/wronggo/service/ContentService.java
package com.wronggo.service;

import com.wronggo.model.PageResult;
import com.wronggo.model.TbContent;

import java.util.List;

public interface ContentService {
    public List<TbContent> findByCategoryId(Long categoryId);
    public void add(TbContent content);
    public void update(TbContent content);
    public void delete(Long[] ids);
    TbContent selectByPrimaryKey(Long id);
    public PageResult findPage(int pageNum, int pageSize);
}
